grape-swagger.gemspec

```diff
@@ -18,6 +17,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.7'
   s.add_runtime_dependency 'grape', '~> 1.3'
 
-  s.files         = `git ls-files`.split("\n")
+  s.files = Dir['lib/**/*', 'CHANGELOG.md', 'CONTRIBUTING.md', 'README.md', 'RELEASING.md', 'UPGRADING.md', 'LICENSE.txt', 'grape-swagger.gemspec']
```

Comment:
wouldn't it be more explicit by excluding the spec dir?

Reply:
<img width="680" alt="Screenshot 2023-10-22 at 14 29 43" src="https://github.com/ruby-grape/grape-swagger/assets/8418467/1cd94e57-66fa-48a9-8b8d-8ff668a6f48f">
I think it's easier to include specific files than rejecting. Yet, bundler [template](https://github.com/rubygems/rubygems/blob/812366203b3fd9bb4e278db790c23f35f947c733/bundler/lib/bundler/templates/newgem/newgem.gemspec.tt#L30-L33) is rejecting folders and files but not all of them. For instance, the Dangerfile would still be there, even the .github folder.

Reply:
No that's just the template applied if you `bundler gem` today